<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>equip</class>
 <widget class="QMainWindow" name="equip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1497</width>
    <height>770</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1404</width>
    <height>0</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>251</width>
      <height>721</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Ajouter /Modifier </string>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>181</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>113</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: rgba(255, 255, 255, 150); /* Semi-transparent white */
    border: 2px solid black; /* Black border */
    border-radius: 10px;
    color: black; /* Default text color */
}

QLineEdit:hover {
    background-color: rgba(255, 255, 255, 180); /* Slightly less transparent on hover */
    border: 2px solid black; /* Keep border black */
    border-radius: 10px;
}

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: italic 9pt &quot;Comic Sans MS&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Id   equipement&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>113</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: rgba(255, 255, 255, 150); /* Semi-transparent white */
    border: 2px solid black; /* Black border */
    border-radius: 10px;
    color: black; /* Default text color */
}

QLineEdit:hover {
    background-color: rgba(255, 255, 255, 180); /* Slightly less transparent on hover */
    border: 2px solid black; /* Keep border black */
    border-radius: 10px;
}

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>440</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>113</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: rgba(255, 255, 255, 150); /* Semi-transparent white */
    border: 2px solid black; /* Black border */
    border-radius: 10px;
    color: black; /* Default text color */
}

QLineEdit:hover {
    background-color: rgba(255, 255, 255, 180); /* Slightly less transparent on hover */
    border: 2px solid black; /* Keep border black */
    border-radius: 10px;
}

</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>440</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>111</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: italic 9pt &quot;Comic Sans MS&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;nom&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>370</y>
       <width>171</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>111</width>
       <height>0</height>
      </size>
     </property>
    </widget>
    <widget class="QDateEdit" name="dd">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>510</y>
       <width>191</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>110</width>
       <height>0</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: italic 9pt &quot;Comic Sans MS&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Type&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>240</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: italic 9pt &quot;Comic Sans MS&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;description&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>340</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: italic 9pt &quot;Comic Sans MS&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;quantite&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>400</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: italic 9pt &quot;Comic Sans MS&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Etat&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>480</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: italic 9pt &quot;Comic Sans MS&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Date   achat&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>650</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>93</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgba(255, 255, 255, 150); /* Semi-transparent white */
    border: 2px solid black; /* Black border */
    border-radius: 8px;
    color: black; /* Default text color */
}

QPushButton:hover {
    background-color: rgba(255, 255, 255, 180); /* Slightly less transparent on hover */
    border: 2px solid black; /* Keep border black */
}
</string>
     </property>
     <property name="text">
      <string>modifier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>93</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgba(255, 255, 255, 150); /* Semi-transparent white */
    border: 2px solid black; /* Black border */
    border-radius: 8px;
    color: black; /* Default text color */
}

QPushButton:hover {
    background-color: rgba(255, 255, 255, 180); /* Slightly less transparent on hover */
    border: 2px solid black; /* Keep border black */
}
</string>
     </property>
     <property name="text">
      <string>effacer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="autoFillButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>580</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>ai auto fill</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>650</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgba(255, 255, 255, 150); /* Semi-transparent white */
    border: 2px solid black; /* Black border */
    border-radius: 8px;
    color: black; /* Default text color */
}

QPushButton:hover {
    background-color: rgba(255, 255, 255, 180); /* Slightly less transparent on hover */
    border: 2px solid black; /* Keep border black */
}
</string>
     </property>
     <property name="text">
      <string>ajouter</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>290</y>
      <width>851</width>
      <height>431</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>851</width>
      <height>411</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>851</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableView {
        border: none;
        gridline-color: black;

      
    }
    QTableView::item { 
        padding: 15px; 
    }
    QHeaderView::section {
        background-color: white;
        padding: 10px;
        font-size: 11px;
        border: none;
       gridline-color: black;
    }
QTableView::item:hover {
      background-color: #d3e0ea;
     color: black;&quot;
    }

    </string>
    </property>
    <property name="gridStyle">
     <enum>Qt::PenStyle::SolidLine</enum>
    </property>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QComboBox" name="trier">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>250</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="recherche">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>190</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
           border: 0.5px solid black;
         border-radius: 15px;
        padding: px;

         color: black;
        }
 QLineEdit:focus {
           border: 1px solid #4A90E2;&quot;  
         }
   </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="supprimer">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>250</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>141</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: 0.5px solid black;
    border-radius: 15px;
    padding: 5px;
    color: black;
}

QLineEdit:focus {
    border: 1px solid #4A90E2; /* Change border color when focused */
    background-color: #f0f8ff; /* Optional: lighter background on focus */
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>-10</y>
      <width>451</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 italic 18pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:48pt; color:#ffffff;&quot;&gt;Equipement&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnemploye">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>161</width>
      <height>0</height>
     </size>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;employe&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 0.5px solid black;
    border-radius: 5px;
    padding: 10px;
    font-weight: bold; /* Texte en gras */
    font-size: 14px;
}

QPushButton:hover,
QPushButton:pressed {
    background-color: #D3D3D3;
    border: 0.5px solid black;
}
</string>
    </property>
    <property name="text">
     <string>Employe</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnpatient">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>161</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 0.5px solid black;
    border-radius: 5px;
    padding: 10px;
    font-weight: bold; /* Texte en gras */
    font-size: 14px;
}

QPushButton:hover,
QPushButton:pressed {
    background-color: #D3D3D3;
    border: 0.5px solid black;
}
</string>
    </property>
    <property name="text">
     <string>Patient</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnvaccins">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 0.5px solid black;
    border-radius: 5px;
    padding: 10px;
    font-weight: bold; /* Texte en gras */
    font-size: 14px;
}

QPushButton:hover,
QPushButton:pressed {
    background-color: #D3D3D3;
    border: 0.5px solid black;
}
</string>
    </property>
    <property name="text">
     <string>Vaccins</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnequipement">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 0.5px solid black;
    border-radius: 5px;
    padding: 10px;
    font-weight: bold; /* Texte en gras */
    font-size: 14px;
}

QPushButton:hover,
QPushButton:pressed {
    background-color: #D3D3D3;
    border: 0.5px solid black;
}
</string>
    </property>
    <property name="text">
     <string>Equipement</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnformation">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 0.5px solid black;
    border-radius: 5px;
    padding: 10px;
    font-weight: bold; /* Texte en gras */
    font-size: 14px;
}

QPushButton:hover,
QPushButton:pressed {
    background-color: #D3D3D3;
    border: 0.5px solid black;
}
</string>
    </property>
    <property name="text">
     <string>Formation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnrecherche">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>550</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 0.5px solid black;
    border-radius: 5px;
    padding: 10px;
    font-weight: bold; /* Texte en gras */
    font-size: 14px;
}

QPushButton:hover,
QPushButton:pressed {
    background-color: #D3D3D3;
    border: 0.5px solid black;
}
</string>
    </property>
    <property name="text">
     <string>Recherche</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logout">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>640</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 0.5px solid black;
    border-radius: 5px;
    padding: 10px;
    font-weight: bold; /* Texte en gras */
    font-size: 14px;
}

QPushButton:hover,
QPushButton:pressed {
    background-color: #D3D3D3;
    border: 0.5px solid black;
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="logo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 0.5px solid black;
    border-radius: 5px;
    padding: 10px;
    font-weight: bold; /* Texte en gras */
    font-size: 14px;
}

QPushButton:hover,
QPushButton:pressed {
    background-color: #D3D3D3;
    border: 0.5px solid black;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pdf">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>110</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pdf_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>110</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>250</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>29</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgba(255, 255, 255, 150); /* Semi-transparent white */
    border: 2px solid black; /* Black border */
    border-radius: 8px;
    color: black; /* Default text color */
}

QPushButton:hover {
    background-color: rgba(255, 255, 255, 180); /* Slightly less transparent on hover */
    border: 2px solid black; /* Keep border black */
}
</string>
    </property>
    <property name="text">
     <string>supprimer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rechercher">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>180</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="historique">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>100</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="actuliser">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>230</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_tri">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>250</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Tri par :</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1497</width>
     <height>26</height>
    </rect>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
